src/api/files.api.js

```diff
@@ -349,10 +350,10 @@ export async function getFileFromUrl(url, options = {}) {
     const { timeout = null, responseType = null } = options
```

Comment:
It seems that getFileFromUrl is only called from the classes in ImportFile/parser. Since they all inherit FileParser, could it go there?

Reply:
We could, it was previously used in multiple unrelated places.
I might want to leave it here, as it is doesn't really have the file parser "touch", and fits quite well in the "File API" file